cln-rpc/src/primitives.rs

```diff
@@ -78,6 +78,10 @@ impl Amount {
     pub fn msat(&self) -> u64 {
         self.msat
     }
+
+    pub fn to_sat(self) -> u64 {
+        self.msat()/1000
```

Comment:
This seems unrelated to the PR at hand. Also this division is not safe as it drops 3 digits. This should rather be a fallible function that fails if msat is not a multiple of 1000

Reply:
Reverted.